orangecontrib/text/corpus.py

```diff
@@ -657,3 +658,20 @@ def arrays_equal(a, b):
                 np.array_equal(self.pos_tags, other.pos_tags) and
                 self.domain == other.domain and
                 self.ngram_range == other.ngram_range)
+
+
+@summarize.register(Corpus)
+def summarize_(corpus: Corpus) -> PartialSummary:
+    """
+    Provides automated input and output summaries for Corpus
+    """
+    table_summary = summarize.dispatch(Table)(corpus)
+    extras = (
+        (
+            f"<br/><nobr>Total tokens: {sum(map(len, corpus.tokens))}, "
+            f"Total types: {len(corpus.dictionary)}</nobr>"
```

Comment:
In my opinion total types are less understandable. @VesnaT what do you think bout replacing it with `Number unique tokes` or something similar.